python/tvm/relax/frontend.py

```diff
@@ -51,11 +52,6 @@ def __init__(self, definition_scope, diag_ctx):
         self.module = {}
         super().__init__()
 
-    def span_to_span(self, span):
```

Comment:
I don't think this code is equivalent to tvm_span_from_synr, we need to be really careful about the string -> source name mapping.